.github/workflows/shared-terraform-chatops.yml

```
@@ -456,40 +463,14 @@ jobs:
-            ||  contains(needs.context.outputs.labels, 'terraform-cloudflare-provider') }}
-        run: |-
-          echo "uses_github=${USES_GITHUB}" >> $GITHUB_OUTPUT
-          echo "uses_opsgenie=${USES_OPSGENIE}" >> $GITHUB_OUTPUT
-          echo "test_needs_aws=${USES_AWS}" >> $GITHUB_OUTPUT
-          echo "uses_spotinst=${USES_SPOTINST}" >> $GITHUB_OUTPUT
-          echo "uses_datadog=${USES_DATADOG}" >> $GITHUB_OUTPUT
-          echo "uses_tfe=${USES_TFE}" >> $GITHUB_OUTPUT
-          echo "uses_cloudflare=${USES_CLOUDFLARE}" >> $GITHUB_OUTPUT
-
+      - uses: actions/create-github-app-token@v1
+        if: ${{ needs.context.outputs.uses_github == 'true' }}
```

actionlint (1.7.4)

property "uses_github" is not defined in object type {atmos_version: string; base: string; base_repo_name: string; base_repo_owner: string; body: string; closed_at: string; created_at: string; found: string; go_mod_path: string; go_sum_path: string; head_repo_name: string; head_repo_owner: string; head_sha: string; json: string; labels: string; merged_at: string; number: string; terraform_state_path: string; terraform_state_retention_days: string; test_aws_role: string; test_needs_aws: string; test_needs_fixtures: string; test_skip_concurrency: string; title: string; url: string}

(expression)